src/stan/services/arguments/arg_variational_adapt_engaged.hpp

```diff
@@ -0,0 +1,26 @@
+#ifndef STAN_SERVICES_ARGUMENTS_ARG_VARIATIONAL_ADAPT_ENGAGED_HPP
+#define STAN_SERVICES_ARGUMENTS_ARG_VARIATIONAL_ADAPT_ENGAGED_HPP
+
+#include <stan/services/arguments/singleton_argument.hpp>
+
+namespace stan {
+  namespace services {
+
+    class arg_variational_adapt_engaged: public bool_argument {
+    public:
+      arg_variational_adapt_engaged(): bool_argument() {
+        _name = "engaged";
+        _description = "Adaptation engaged?";
+        _validity = "[0, 1]";
+        _default = "1";
+        _default_value = true;
```

Comment:
I think if we say the default is "1", then the default value should be 1, not true.  Or we should change the way the default's written.  This is again probably copied from other examples which should also be changed.